### DISCUSSION TITLE
Could it be possible to train a LoRA for ControlNet?

### DISCUSSION THREAD

**Discussion by OedoSoldier** [lllyasviel/ControlNet#393] [Ideas]
I had this idea after seeing the impressive results of the tile model. It occurred to me that if we could train a LoRA model for the tile model, we might be able to achieve a form of style transfer, or something else, like fine-tuning the tile model based on instruct pix2pix dataset?🤔

**Comment by PrivateMaRyan** [comment]
have u got the answer? I wonder how to do it, too

**Reply by crapthings** [reply]
https://huggingface.co/stabilityai/control-lora
https://github.com/HighCWu/ControlLoRA
https://github.com/HighCWu/control-lora-v2
